### ISSUE TITLE
unified: on update, the port forwarding fails

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-web#3024]
## Describe the bug

After updating Accessibility Insights for Android (prod) to 2020.702.3, port forwarding fails on first launch.

**To Reproduce**

1. open Accessibility Insights for Android
2. update to 2020.702.3
3. connect a device
4. port forwarding fails

## Expected behavior

port forwarding succeeds

## Context (please complete the following information)

- OS Name & Version:  Windows
- AI-Web Version & Environment:  2020.702.3

## Are you willing to submit a PR?
No 

## Did you search for similar existing issues?
Yes

## Additional context

this seems to be a consistent repo

**Comment by karanbirsingh** [comment]
In the cases you've seen, was this on a device/emulator, or both? I haven't been able to get a repro on my emulator @ferBonnin

**Comment by ferBonnin** [comment]
@karanbirsingh emulator for me.
@LiLoDavis were you using an emulator or physical device?

**Comment by LiLoDavis** [comment]
> @LiLoDavis were you using an emulator or physical device?

I was using a physical device.

**Comment by karanbirsingh** [comment]
worked offline with Fer to look into this & we were unable to repro the issue on other devices; we decided to close for now
